program/plugins/nikto_domino.plugin

```diff
@@ -54,27 +54,28 @@ sub nikto_domino {
 
     # quit if banner is not from IBM/Lotus Domino
     if (($mark->{'banner'} !~ /lotus-?domino/i) && ($response->{'server'} !~ /lotus-?domino/i) && ( $res ne "200" && $content !~ /src="\/domcfg.nsf\// ) ) {
-	$is_domino=0;
-	}
+        $is_domino=0;
+    }
     else {
```

Comment:
Ahh, sounds reasonable. Proposal for an updated detection in https://github.com/sullo/nikto/pull/402